apps/dashboard/app/javascript/xdmod/jobs.js

```diff
@@ -77,15 +79,17 @@ function table(context, helpers) {
   // <table class="table table-sm table-striped table-condensed">
   tableElement.classList.add('table', 'table-sm', 'table-striped', 'table-condensed');
 
-  thead = document.createElement('thead');
+  const thead = document.createElement('thead');
+  // Empty th to accommodate for the job analytics button
   thead.innerHTML = '<tr> \
-                      <th>ID</th> \
-                      <th>Name</th> \
-                      <th>Date</th> \
-                      <th>CPU</th> \
+                      <th class="sr-only">Analytics Toggle</th> \
+                      <th class="id">ID</th> \
+                      <th class="name">Name</th> \
+                      <th class="date">Date</th> \
+                      <th class="sr-only">Analytics</th> \
```

Comment:
Just to be clear - the button is in the 1st column and that expands that data in the 5th column?

I think there's still something off about this. The 5th column reads as `blank` if you've ever expanded it before and collapsed it. And if you've never collapsed it, it skips the column entirely, when it should read something like `analytics column <something about needing to expand it>`.

That said - I can merge this now and pick it up later just because this has dragged on for a while.

Reply:
> Just to be clear - the button is in the 1st column and that expands that data in the 5th column?

Yes , that is the solution I came up with to display the analytics without creating a new row.